src/openapi_server/controllers/search_controller.py

```diff
@@ -0,0 +1,36 @@
+from math import ceil
+
+import connexion
+
+from openapi_server.models import SearchResult
+from openapi_server.models.search_query import SearchQuery  # noqa: E501
+from openapi_server.models.search_results import SearchResults  # noqa: E501
+from wrappers import cobs
+from wrappers.cobs import COBS_TERM_SIZE
+
+
+def search_post(search_query=None):  # noqa: E501
+    """search_post
+
+     # noqa: E501
+
+    :param search_query: 
+    :type search_query: dict | bytes
+
+    :rtype: SearchResults
+    """
+    if connexion.request.is_json:
+        search_query = SearchQuery.from_dict(connexion.request.get_json())  # noqa: E501
+
+    results = cobs.search(search_query.seq, search_query.threshold)
+    num_kmers = ceil(len(search_query.seq) / COBS_TERM_SIZE)
```

Comment:
It should be `num_kmers = len(search_query.seq) - COBS_TERM_SIZE + 1`.

Reply:
I don't understand, isn't that number of base letters, not number of k-mers (k == term_size)?

Reply:
I can refer you to this page for how kmers are derived from a sequence: https://en.wikipedia.org/wiki/K-mer